scripts/upload-source-maps.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import { execSync } from 'child_process'
+
+const main = () => {
+	if (process.env.VERCEL_ENV === 'development') {
+		return
+	}
```

Comment:
🧪 I feel like it could be good to have a way to confirm that the basics of source maps are working locally... maybe when `VERCEL_ENV` is `development`, we could take on the optional [`--dry-run` parameter](https://github.com/DataDog/datadog-ci/tree/master/src/commands/sourcemaps#upload) and leave everything else the same?

Reply:
what do you mean by working locally? that we can test uploading them to datadog locally?

Reply:
More just to see that `datadog-ci` picks up on the `*.map` files, and that they're removed after the script runs! I believe the `--dry-run` flag will mean source maps aren't uploaded. Definitely feels optional 👍